Increase the maximum-allowed user name length to match the development versions.
--- krb5-1.2.7/src/lib/krb5/os/kuserok.c	2002-12-04 14:22:40.000000000 -0500
+++ krb5-1.2.7/src/lib/krb5/os/kuserok.c	2002-12-04 14:22:33.000000000 -0500
@@ -38,7 +38,7 @@
 #define getpwnam(user) getpwnam((char *)user)
 #endif
 
-#define MAX_USERNAME 10
+#define MAX_USERNAME 65
 
 /*
  * Given a Kerberos principal "principal", and a local username "luser",
